app/assets/javascripts/application.js

```diff
@@ -33,4 +33,16 @@ $(function() {
   $('[data-behaviour~=date-and-time-picker]').datetimepicker({
     format: 'YYYY/MM/DD hh:mm'
   });
-});
+
+  if ($(window).width() <= 560 ) {
+  $('#column2 .volunteer-ops-search').prependTo('#column1 .row');
```

Comment:
because you're getting this element in multiple lines you can have it be a variable something like this `const volunteerSearch = $('#column2 .volunteer-ops-search')` and put it on line 36

Reply:
You right @Kachulio1 I thought about it, but I'm not very efficient with js.